cmd/soroban-cli/src/commands/config/identity/show.rs

```diff
@@ -13,15 +13,17 @@ pub enum Error {
     StrKey(#[from] stellar_strkey::DecodeError),
 }
 
-#[derive(Debug, clap::Args)]
-#[group(skip)]
```

Comment:
So, this will add an arggroup for these?

Reply:
Oops added them back.